src/account-abstraction.md

```diff
@@ -0,0 +1,168 @@
+<h1>
+    Account Abstraction 
+</h1>
+
+
+Account Abstraction is a win for usability in web3 and on Starknet it has been supported from the beginning at the protocol level. For a smart contract to be considered an account contract it must at least implement the interface defined by SNIP-6. Additional methods might be required for advanced account functionality
+
+
+
+Account Abstraction (AA) replaces Externally Owned Accounts (EOA) with smart wallets, which are implemented as smart contracts. Each smart wallet contract must implement the SNIP-6 interface. This transformation allows for greater flexibility and customization compared to traditional EOAs.
```

Comment:
It's not really a "transformation" but more an improvement. You can just remove the transformation word.